### PR TITLE
New YOLOv5 support

### PR DESCRIPTION
**Pull request by Jacobsolawetz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +69,17 @@ python clip_object_tracker.py --source data/video/fish.mp4 --url https://detect.
 
 **NOTE you must provide a valid API key from [Roboflow](docs.roboflow.com)
 
+###Run with Torch Hub Yolov5
+This method allows you to use the latest Yolov5 version found on [Torch Hub](https://pytorch.org/hub/ultralytics_yolov5/). The `github` flag sets the model path on torch hub which is `ultralytics/yolov5` by default. The `type` flag sets the type of model like yolov5s, yolov5l, custom, etc. The default is `yolov5s`. The `sourceType` flag determines where to load the weights from which is either local or github. The default is local which means you have to provide pretrained weights.
+```bash
+
+python clip_object_tracker.py --weights models/yolov5s.pt --source "data/video/cars.mp4" --detection-engine hub --github ultralytics/yolov5 --type yolov5s --sourceType github --info
```
Comment (Jacobsolawetz) [inline README.md:76]: testing custom models and posting ex here

Comment (Jacobsolawetz) [reply README.md:76]: https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data

**Comment by Jacobsolawetz** [comment]
@maxhs2014 any idea where we were on this?

**Comment by maxhs2014** [comment]
@Jacobsolawetz last I checked this worked perfectly fine but models trained on the older version of yolov5 in our notebooks before did not work because of conflicting yolov5 code. Now that we are completely on a suitable version of yolov5 it should work fine (I can test again), but the same thing could happen in the future if roboflow notebook models are on an older version than torch hub.

**Pull request closed without merging**